config/initializers/devise.rb

```diff
@@ -308,4 +308,15 @@
   # When set to false, does not sign a user in automatically after their password is
   # changed. Defaults to true, so a user is signed in automatically after changing a password.
   # config.sign_in_after_change_password = true
+
+  config.jwt do |jwt|
+    jwt.secret = Rails.application.credentials.fetch(:secret_key_base)
```

Comment:
- [ ] Where are we supposed to get this value locally? Your app is starting because of this. The DB can not be created. You would add those specification in your readme file to show how to do.